tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/TaskProblemAction.java

```diff
@@ -59,9 +60,9 @@ public void run( final TaskProblem taskProblem, final ISelectionProvider provide
     {
         final IFile file = MigrationUtil.getIFileFromTaskProblem( taskProblem );
 
-        new Job( "Marking migration problem as done" )
+        new UIJob( "Marking migration problem as done" )
```

Comment:
There is nothing in that job that needs the UI thread except for what is in the UIUtil.async() call.